battle/moves/move_effects.asm

```diff
@@ -1387,10 +1229,6 @@ Whirlpool:
 	endmove
 
 SuperFang:
```

Comment:
What is the reason for leaving labels like SuperFang and Psywave?  There are a few other cases like that in this file.

Reply:
Because while they share the same effect pointer, they use a different effect ID. It was either this or adding ` ; EFFECT_SUPER_FANG` behind the appropiate pointer. I thought this looked reasonably good.